AnyText/AnyText/Grammars/AnyTextGrammar.manual.cs

```diff
@@ -138,6 +138,16 @@ public partial class KeywordRule
 
         }
 
+        public partial class RuleNameIDRule
+        {
+            public override bool IsDefinition => true;
+        }
+
+        public partial class IDRule
+        {
+            public override bool IsReference => true;
```

Comment:
die IDRule ist hierfür eigentlich die falsche. Das müsste eigentlich in der `RuleReferenceExpressionRuleIDRule` oder so stehen, bzw. eigentlich kann man das schon generisch in der `ResolveRule` überschreiben